showcase/src/app/app.component.html

```diff
@@ -67,15 +67,27 @@
 								</button>
 							</div>
 							<div class="stark-app-bar-content-right-actions-alt">
-								<button mat-mini-fab [matTooltip]="'SHOWCASE.STYLEGUIDE.TITLE' | translate">
-									<mat-icon starkSvgViewBox svgIcon="television-guide"></mat-icon>
+								<a href="https://github.com/NationalBankBelgium" target="_blank">
+									<button mat-mini-fab mat-button class="mat-icon-button mat-accent"
+											[matTooltip]="'STARK.ICONS.GITHUB' | translate">
+									<mat-icon class="custom-icon" svgIcon="github-circle">
```

Comment:
For the Github icon, I think we can make it bigger so we can clearly see the logo. What do you think?

For that you need to add this CSS:

```css
.stark-app-bar-content-right-actions-alt .mat-icon-button.some-class-xxxxxxx {
  background-color: #000;   /* Github logo with black background */
  
  .mat-button-wrapper {
    padding: 0;
  }
  mat-icon {
    height: 40px;
    width: 40px;
  }
}
```

And I think we can shuffle the buttons like this:
![showcase fab buttons](https://user-images.githubusercontent.com/5698318/48060244-bf55b800-e1bb-11e8-9db9-444d5f71be45.PNG)

So this means that the HTML should be change like this:

```html
<div class="stark-app-bar-content-right-actions-alt">
	<a href="https://stark.nbb.be/api-docs/stark-ui/latest/" target="_blank">
		<button mat-mini-fab mat-icon-button color="success"
				[matTooltip]="'STARK.ICONS.STARK_UI' | translate">
			<mat-icon class="custom-icon" svgIcon="television-guide" starkSvgViewBox>
			</mat-icon>
		</button>
	</a>
	<a href="https://stark.nbb.be/api-docs/stark-core/latest/" target="_blank">
		<button mat-mini-fab mat-icon-button color="primary"
				[matTooltip]="'STARK.ICONS.STARK_CORE' | translate">
			<mat-icon class="custom-icon" svgIcon="atom" starkSvgViewBox>
			</mat-icon>
		</button>
	</a>
	<a href="https://github.com/NationalBankBelgium" target="_blank">
		<button mat-mini-fab mat-icon-button class="some-class-xxxxxxx"
				[matTooltip]="'STARK.ICONS.GITHUB' | translate">
			<mat-icon class="custom-icon" svgIcon="github-circle" starkSvgViewBox>
			</mat-icon>
		</button>
	</a>
</div>
```